mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -719,36 +719,41 @@ public void runVoid() throws MailboxException {
         );
     }
 
-    private AnnotationMapper getAnnotationMapper(MailboxPath mailboxPath, MailboxSession session)
+    private MailboxId getMailboxId(MailboxPath mailboxPath, MailboxSession session)
             throws MailboxException, MailboxNotFoundException {
         MailboxMapper mailboxMapper = mailboxSessionMapperFactory.getMailboxMapper(session);
         Mailbox mailbox = mailboxMapper.findMailboxByPath(mailboxPath);
-        AnnotationMapper annotationMapper = mailboxSessionMapperFactory.getAnnotationMapper(mailbox.getMailboxId(), session);
-        return annotationMapper;
+
+        return mailbox.getMailboxId();
```

Comment:
You should be able to use this.getMailbox(mailboxPath, session).getId() instead, no?